packages/applications/ssr/src/components/molecules/FiltersTagList.tsx

```diff
@@ -55,6 +56,19 @@ export const FiltersTagList: FC<FiltersTagListProps> = ({ filters }) => {
       return router.push(url);
     }
   };
+  const onDelete = () => {
+    const newSearchParams = new URLSearchParams();
+
+    if (searchBarParams) {
+      const existingBarSearchValue = searchParams.get(searchBarParams);
+      if (existingBarSearchValue) {
+        newSearchParams.set(searchBarParams, existingBarSearchValue);
+      }
+    }
+
+    const url = `${pathname}?${newSearchParams.toString()}`;
```

Comment:
je suppose qu'il faut ca non ?
```suggestion
    const url =  newSearchParams.length ? `${pathname}?${newSearchParams.toString()}` : pathname;
```